plonk/src/proof_system/snark.rs

```diff
@@ -76,7 +76,7 @@ where
         let domain_size = circuit.eval_domain_size()?;
         let num_inputs = circuit.num_inputs();
         let supported_hiding_bound = 2;
```

Comment:
Is `supported_hiding_bound` still used in the function? We can remove it if not.